server/converters/cxgtool.py

```diff
@@ -409,6 +412,7 @@ def save_metadata(container, metadata):
     with tiledb.DenseArray(a_name, mode="w") as A:
         A.meta["cxg_version"] = CXG_VERSION
         A.meta["cxg_properties"] = json.dumps(metadata)
+        A.meta["colors"] = json.dumps(anndata_colors_to_cxg_colors(adata))
```

Comment:
there is no error handling here, and the `anndata_colors_to_cxg_colors` function just throws undifferentiated Exception objects when it gets bad data.

It seems very likely that this particular interface is going to be very fragile as humans will be poking values into it regularly.   It needs more error handling/reporting, if not an option to ignore the values/errors and convert without colors.